src/foundry/client/data/documents/chat-message.d.ts

```diff
@@ -253,6 +236,20 @@ declare global {
       alias?: string | undefined;
     }
 
+    interface SpeakerData {
+      /** The Scene in which the speaker resides */
+      scene: InstanceType<ConfiguredDocumentClass<typeof Scene>> | null;
```

Comment:
Since you already replaced that with direct references in other PRs, I don't think we need `ConfiguredDocumentClass` and siblings anymore.

Reply:
Updated 🤞